src/new_game.c

```diff
@@ -222,3 +225,11 @@ static void ResetItemFlags(void)
     memset(&gSaveBlock3Ptr->itemFlags, 0, sizeof(gSaveBlock3Ptr->itemFlags));
 #endif
 }
+
+static void ResetDexnav(void)
```

Comment:
```suggestion
static void ResetDexNav(void)
```
and associated changes elsewhere